aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/helpers/CognitoDeviceHelperTests.kt

```diff
@@ -0,0 +1,29 @@
+package com.amplifyframework.auth.cognito.helpers
+
+import kotlin.test.assertEquals
+import org.junit.Test
+
+class CognitoDeviceHelperTests {
+
+    @Test
+    fun testHex() {
```

Comment:
nit: is `test` prefix required? Previous ver of JUnit needed it. But I don't think it is necessary anymore.
How about naming this method as in what it tests
generated_hex_is_correct or sth like that.